connection-service/new-vm-join-domain/new-vm-join-domain.json

```diff
@@ -88,15 +82,14 @@
         "imagePublisher": "MicrosoftWindowsServer",
         "imageOffer": "WindowsServer",
         "windowsOSVersion": "2016-Datacenter",
-        "apiVersion": "2015-06-15",
         "nicName": "[concat(parameters('dnsLabelPrefix'),'Nic')]",
         "updateipTemplateUri": "[concat(parameters('_artifactsLocation'), '/update-nic.json', parameters('_artifactsLocationSasToken'))]"
     },
     "resources": [
         {
             "type": "Microsoft.Network/networkInterfaces",
             "name": "[variables('nicName')]",
-            "apiVersion": "[variables('apiVersion')]",
```

Comment:
Yeah I agree it seems better to have the individual API dates we want than some global variable that always seems wrong...